detection_rules/integrations.py

```diff
@@ -106,3 +222,64 @@ def get_integration_manifests(integration: str) -> list:
     print(f"loaded {integration} manifests from the following package versions: "
           f"{[manifest['version'] for manifest in manifests]}")
     return manifests
+
+
+def get_integration_schema_data(data, meta) -> Generator[dict, None, None]:
+    """Iterates over specified integrations from package-storage and combines schemas per version."""
+
+    # lazy import to avoid circular import
+    from .rule import (  # pylint: disable=import-outside-toplevel
+        QueryRuleData, RuleMeta, TOMLRuleContents)
+
+    data: QueryRuleData
+    meta: RuleMeta
```

Comment:
is this more correct

```suggestion
    data: QueryRuleData = data
    meta: RuleMeta = meta
```